src/sourdough-bread-with-seeds-and-grains.md

```diff
@@ -0,0 +1,47 @@
+# Sourdough bread with seeds and grains
+
+![Sourdough bread with seeds and grains](pix/sourdough-bread-with-seeds-and-grains.webp)
+
+- ⏲️ Prep time: 15 min
+- 🍳 Cook time: 120 min
+
+## Ingredients
+
+**For the preparation:**
```

Comment:
```suggestion
### For the preparation
```

We don't really have a standard for it, but I've seen many people use the `###` for subheadings under ingredients, so let's stick with that

Reply:
Yeah, we should define that somewhere. I looked into some recipes and did not find one with subsections. I will change it to the `###` notation.